docs/data/sdks/sdk-architecture.md

```diff
@@ -0,0 +1,368 @@
+---
+title: SDK Architecture
+description: Overview to Amplitude Analytics SDK architecture and common usage
+---
+
+This article helps you remove confusion around various Amplitude Analytics SDKs and introduces their architecture, common methods and benefits.
```

Comment:
This article helps you remove confusion around various Amplitude Analytics SDKs and introduces the architecture, common methods, and benefits of the latest SDKs.
```suggestion
This article helps you remove confusion around various Amplitude Analytics SDKs and mainly focuses on introducing the architecture, common methods, and benefits of the latest SDK structure.
```
I think the doc is mainly focused on the architecture, common methods, and benefits not on all various SDKs.